### PR TITLE
Add table for routes that should be implemented

### PR DESCRIPTION
**Pull request by chris-tbd**
This pull request includes an update to the `anatomy-of-a-pfi.mdx` file in the `site/docs/tbdex/pfi` directory. The change adds a table that lists the required routes to be implemented in the main server entry point. The table includes the HTTP method, endpoint, and a description for each route.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206673189382336

### CODE REVIEW

File: site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx [hunk 1]
```diff
@@ -25,6 +25,40 @@ Your main server entrypoint should fulfill the following responsibilities:
 * Accept `GET` requests for Offerings and Exchanges data
 * Handle submission requests for new `RFQ`, `Order`, and `Close` messages
 
+Here is a list of the reequired routes you'll need to implement:
```
Comment (angiejones) [inline site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx:28]:
```suggestion
Here is a list of the required routes you'll need to implement:
```

File: site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx [hunk 2]
```diff
@@ -25,6 +25,40 @@ Your main server entrypoint should fulfill the following responsibilities:
...
+  <tbody>
+    <tr>
+      <td>POST</td>
+      <td>/exchanges</td>
+      <td>Used to send an RFQ aka create an exchange.</td>
+    </tr>
+    <tr>
+      <td>PUT</td>
+      <td>/exchanges/:id</td>
+      <td>Used to add additional messages to a pre-existing exchange e.g. order or close</td>
```
Comment (angiejones) [inline site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx:47]:
```suggestion
      <td>Used to add additional messages (e.g. `Order`, `Close`) to an existing exchange</td>
```

File: site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx [hunk 3]
```diff
@@ -25,6 +25,40 @@ Your main server entrypoint should fulfill the following responsibilities:
...
+      <th>Method</th>
+      <th>Endpoint</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>POST</td>
+      <td>/exchanges</td>
+      <td>Used to send an RFQ aka create an exchange.</td>
```
Comment (angiejones) [inline site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx:42]:
```suggestion
      <td>Used to send an RFQ aka create an exchange</td>
```

File: site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx [hunk 4]
```diff
@@ -25,6 +25,40 @@ Your main server entrypoint should fulfill the following responsibilities:
...
+    </tr>
+    <tr>
+      <td>PUT</td>
+      <td>/exchanges/:id</td>
+      <td>Used to add additional messages to a pre-existing exchange e.g. order or close</td>
+    </tr>
+    <tr>
+      <td>GET</td>
+      <td>/exchanges</td>
+      <td>Used to get exchanges</td>
```
Comment (angiejones) [inline site/docs/tbdex/pfi/anatomy-of-a-pfi.mdx:52]:
```suggestion
      <td>Used to get the exchange IDs of all exchanges that match a specified filter</td>
```

**Review by angiejones — CHANGES_REQUESTED** [review]
Kotlin tests failing due to compilation error:
https://github.com/TBD54566975/developer.tbd.website/actions/runs/8426165576/job/23073899230?pr=1352#step:7:44

**Comment by angiejones** [comment]
> Kotlin tests failing due to compilation error: https://github.com/TBD54566975/developer.tbd.website/actions/runs/8426165576/job/23073899230?pr=1352#step:7:44

Since this was causing everyone's PRs to fail, I deployed a fix
https://github.com/TBD54566975/developer.tbd.website/pull/1355

**Comment by chris-tbd** [comment]
> Kotlin tests failing due to compilation error: https://github.com/TBD54566975/developer.tbd.website/actions/runs/8426165576/job/23073899230?pr=1352#step:7:44

@angiejones could you re-review now that those are taken care of? And thank you for addressing that!

**Review by angiejones — APPROVED** [review]

**Comment by chris-tbd** [comment]
@angiejones would you mind merging? I am currently restricted from merging

**Pull request merged**